### ISSUE TITLE
Auto placement of green/red flag

### ISSUE THREAD

**Issue by Krazyfuq** [ScreepsOCS/screeps.behaviour-action-pattern#607]
I am trying but haven't been able to figure out what is triggering this flag placement, I have had this in one room with this flag for days now and just don't see why the room needs a pioneer, I keep removing the flag and it keeps returning. Could someone explain what is triggering this?

**Comment by whardeman** [comment]
Task.pioneer is causing that. I've commented out the offending section of `mod.handleRoomDied`
```
    // check if room has a pioneer flag
    let pos = new RoomPosition(25, 25, room.name);
    let flag = FlagDir.find(FLAG_COLOR.claim.pioneer, pos, true);
    if( !flag ){
        room.createFlag(pos, null, FLAG_COLOR.claim.pioneer.color, FLAG_COLOR.claim.pioneer.secondaryColor);
    }
```
These lines are just the part that creates the Pioneer flag if handleRoomDied runs. It's seems to be bugged, because I and other people are also getting it.

**Comment by whardeman** [comment]
#560

**Comment by whardeman** [comment]
Duplicate of #517

**Comment by whardeman** [comment]
Possibly resolved by #589

**Comment by Krazyfuq** [comment]
I just read #517 and think cyberblast is correct, the room I see the flags in have no need for workers so the moment the hauler dies and time it takes to respawn is likely when the flag is placed and the pioneer is spawned. 

thanks whardman
